Tools/Scripts/run-jsc-stress-tests

```diff
@@ -1118,6 +1118,7 @@ BASE_MODES = [
             "--useSamplingProfiler=true",
             "--airForceIRCAllocator=true",
             "--useDataICInOptimizingJIT=true",
+            "--forceUnlinkedDFG=true",
```

Comment:
Do we really want to start running tests? Might be limiting in future patches that don't fix all issues

Reply:
I would like to keep things working while landing a patch so I expect that this option is OK. And we can disable it anytime if we think that's better.